plugins/scaffolder-node/src/actions/createTemplateAction.test.ts

```diff
@@ -52,18 +55,18 @@ describe('createTemplateAction', () => {
     const action = createTemplateAction({
       id: 'test',
       schema: {
-        input: z =>
-          z.union([
-            z.object({
-              repoUrl: z.string(),
+        input: (zImpl: typeof z) =>
```

Comment:
Writing the type here is not necessary as TypeScript can figure out the underlying type. I believe it was also the intention of using the function form of `z => schema`. If every subsequent usage at the plugin module needs `zImpl: typeof z`, it would have been better to keep it as `schema` in the first place.